awx/ui_next/src/screens/Credential/CredentialAdd/CredentialAdd.jsx

```diff
@@ -44,18 +59,17 @@ function CredentialAdd({ me }) {
           inputs: nonPluginInputs,
           ...remainingValues,
         });
-        const inputSourceRequests = [];
-        Object.entries(pluginInputs).forEach(([key, value]) => {
-          inputSourceRequests.push(
+
+        await Promise.all(
```

Comment:
at first i was thinking it'd be better to concat plugin/non-plugin together, but I think the way useCallback and await works...it'll still finish all this before returning that it's finished